server/tests/test_restart.py

```diff
@@ -45,3 +51,27 @@ def test_manual_channels_same_after_restart(
         channels_end = cf_client.find(name="*")
         assert len(channels_begin) == len(channels_end)
         channels_match(channels_begin, channels_end, PROPERTIES_TO_MATCH + ["test_property"])
+
+
+def check_connection_active(cf_client: ChannelFinderClient) -> bool:
+    try:
+        cf_client.find(name="*")
+        return True
+    except Exception:
+        return False
```

Comment:
Stylistic, but prefer

```suggestion
    try:
        cf_client.find(name="*")
    except Exception:
        return False
    return True
```